apps/judicial-system/backend/src/app/formatters/pdf.ts

```diff
@@ -242,6 +251,13 @@ export async function generateRulingPdf(
       lineGap: 6,
       paragraphGap: 0,
     })
+    .text(
+      `Verjandi sakbornings: ${
+        isFalsy(existingCase.defenderName)
+          ? 'Hefur ekki verið skráður.'
+          : existingCase.defenderName
+      }`,
+    )
```

Comment:
Why add the defender here? The defender is listed in the courtAttendees just above.

Reply:
Not if the defender is not set. I've changed this to say nothing if the defender is set and appears in the courtAttendees and "Hefur ekki verið skráður" if not. Does that make sense to you?

Reply:
Not really. You don't know if courtAttendees includes a defender because it is s free text field. I think the ticket in Asana only specified that the prosecutor request should include the defender. The ruling is a different matter. Ask Anna or Baldur perhaps?

Reply:
Ah! I removed this code. Let's have tensions come up in review. Sounds good?